.travis.yml

```diff
@@ -3,5 +3,6 @@ git:
   depth: 2
 language: node_js
 node_js:
+  - 'iojs'
```

Comment:
For consistency in one file. `node_js` is not quoted.

[BrowserSync/grunt-browser-sync repo uses this style](https://github.com/BrowserSync/browser-sync/blob/master/.travis.yml).